### DISCUSSION TITLE
Is GalaxyBrain achievement broken?

### DISCUSSION THREAD

**Discussion by Th3S4mur41** [community/community#44105] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

I have several replies to discussions marked as answers, but still not getting the Galaxy Brain achievement.

https://github.com/orgs/atos-parallel/discussions/3
https://github.com/orgs/atos-parallel/discussions/4
https://github.com/orgs/atos-parallel/discussions/1
https://github.com/orgs/atos-parallel/discussions/6
https://github.com/orgs/atos-parallel/discussions/7

https://github.com/MH-Photo/community/discussions/1
https://github.com/MH-Photo/community/discussions/2

Is the achievement maybe broken?

**Comment by melvincwng** [comment, marked as answer]
@Th3S4mur41, check out the answer below which I've provided another user previously:  #18467 

In summary, can you check if you **did not** mark any of your own questions/discussions as answers. Because if you do that, you **will not** be able to achieve the Galaxy brain badge. 

That's one of the most common reason I see other github users who previously were facing the same issue as you. In short, they had marked some of their replies as answers in their own discussion threads and as a result, this prevented them from getting the badge. Discussions need to be marked as answer by **other users** for the badge to be unlocked.

Solution would be you would need to unmark those discussions of your own (if there is). And then answer another user's question & have it marked by them.

If all fails, you can contact support. It could really be a bug which I may not be aware of. But I can see from previous discussion threads, you have already logged a  ticket. The support engineers usually reply within a few days to a few weeks, depending on the number of tickets & staffing availability. 

Hope this clarifies and answers your question. 

Take care, and all the best in your achievement hunting journey 😃

**Comment by daniel-cues** [comment]
Hum, I don't know, mark this as answer and I'll check 😉

**Comment by Th3S4mur41** [comment]
Thanks @melvincwng I knew that self answered questions didn't count (no surprise here), but I wasn't aware that they would hinder somebody to get the badge at all (weird limitation or maybe still a bug)...

I did indeed found 3 instances where I answered my own questions:
https://github.com/storybookjs/storybook/discussions/17646
https://github.com/storybookjs/storybook/discussions/18437
https://github.com/semantic-release/semantic-release/discussions/2562

Unmarked them... Let's see what happens when I answer the next question 🥁

**Reply by melvincwng** [reply]
@Th3S4mur41, glad to have helped!

Just remember if you intend to continue achieving the different variations of the Galaxy Brain badge (there's 4 of them - each of which is answered at 2, 8, 16, 32 discussions answered), please do not mark your own answers as it might hinder the process.

Hope this has been useful, good luck!

**Comment by S1i** [comment]
If you delete the repository that you have comments marked as the answer that you created does it still count as your answers or what happens? I have deleted repositories that I have made a discussion on and marked my own comment as the answer and don't know how to fix this.

**Comment by shubheshswain91** [comment]
To get the Galaxy Brain achievement badge, you need to fulfill the following criteria:

Discussions should be in a public repository
Get marked as answers to 2 or more questions
The questions cannot be asked by you. In short, you cannot answer your own questions, those do not count.
In fact, answering your own questions could prevent the achievement from displaying in your profile page.

**Comment by github-staff** [comment]
Because of an influx of users creating low quality, nonsensical or spam posts in the Community with the aim of earning certain Achievements, we’ve made the difficult decision to disable the ability to earn Achievements in our Community.

Note that this change does not affect your ability to earn badges in other projects and communities across GitHub. This change only applies to GitHub’s Community Discussions.

You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Let us know if you have any other questions 🚀
